src/bin/owi.ml

```diff
@@ -179,6 +179,15 @@ let opt_cmd =
   in
   Cmd.v info Term.(const Cmd_opt.cmd $ debug $ unsafe $ files)
 
+let rac_cmd =
+  let open Cmdliner in
+  let info =
+    let doc = "Perform runtime assertion checking" in
```

Comment:
I believe it should be something like `Generate an instrumented file with runtime assertion checking coming from Weasel specifications.` ?